.env.example

```
@@ -1,14 +1,26 @@
 
+# Environment variables for Next.js.
+# Copy this file to .env and fill in the values.
+#
+# NEVER commit .env — it contains secrets and is gitignored.
+
+NEXTJS_ENV
+
 # Resend API key for email sending
 NEXT_PUBLIC_resend
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The NEXT_PUBLIC_resend key should be with a value or have an equal sign

(KeyWithoutValue)